easybuild/easyconfigs/c/CMake/CMake-3.6.1.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'ConfigureMake'
+
+name = 'CMake'
+version = '3.6.1'
+
+homepage = 'http://www.cmake.org'
+description = """CMake, the cross-platform, open-source build system.
+ CMake is a family of tools designed to build, test and package software."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['http://www.cmake.org/files/v%(version_major_minor)s']
+sources = [SOURCELOWER_TAR_GZ]
+
+configopts = '-- -DCMAKE_USE_OPENSSL=1'
+
+# Don't depend on anything, since we will use this to build other software
+# which might want different versions of the same dependency, eg: ncurses
+dependencies = []
```

Comment:
@boegel Do we need these three lines (17 to 19) for a purpose? I can take them out if they're not needed for standardisation and/or information.